### PR TITLE
Security upgrade of node

### PR DESCRIPTION
**Pull request by uladzimirdev**
https://nodejs.org/en/blog/vulnerability/june-2023-security-releases

use `fnm install` or `nvm install` or just install node 18.16.1 manually

### CODE REVIEW

**Comment by ariya** [comment]
> use `fnm install` or `nvm install` or just install node 18.16.1 manually

A notable compelling alternative: https://volta.sh/.

File: .nvmrc
Comment (WiNloSt) [inline .nvmrc:1]: I'm using `nvm` so I'm fine with this. I saw [you made a comment](https://metaboat.slack.com/archives/C010L1Z4F9S/p1687269111470539?thread_ts=1687268735.540199&cid=C010L1Z4F9S) about using `.node-version` too. I'm just not sure if `.nvmrc` will work with most popular node version managers out there.

Comment (uladzimirdev) [reply .nvmrc:1]: https://github.com/shadowspawn/node-version-usage#supporting-products

see the last line in the section, `nvm` - I presume one of the most popular version managers - doesn't support `.node-version`

Comment (WiNloSt) [reply .nvmrc:1]: would it be possible to have 2 files at the same time then? i.e. having both `.nvmrc` and `.node-version`, I think that'll be too much to keep them in sync I guess.

Comment (uladzimirdev) [reply .nvmrc:1]: it doesn't make any sense. `nvmrc` is more or less a standard, `.node-version` is not supported by nvm and `volta`, so it's not a standard yet, but having `nvmrc` simplifies upgrade to another node version at github actions and locally if you use nvm or fnm

**Review by WiNloSt — APPROVED** [review]
running `yarn build-hot` and `yarn test-unit-watch` still work fine locally.

**Pull request merged**